src/ralph/settings.py

```diff
@@ -312,6 +312,7 @@
 FISHEYE_PROJECT_NAME = ""
 VMWARE_USER = None
 VMWARE_PASSWORD = None
+LDAP_SERVER_OBJECT_USER_CLASS = "person" # possible values: "person, user"
```

Comment:
you shouldn't change default value which was 'user' - please set it to 'user' and overwrite it in your own settings if you need 'person'